batch/batch/front_end/front_end.py

```diff
@@ -2152,17 +2198,20 @@ async def ui_get_job(request, userdata, batch_id):
     return await render_template('batch', request, userdata, 'job.html', page_context)
 
 
+# This should really be the exact same thing as the REST endpoint
 @routes.get('/batches/{batch_id}/jobs/{job_id}/log/{container}')
```

Comment:
I don't think this endpoint is even used at all. You can probably just delete it.

Reply:
this is used for downloading logs from the UI. I meant to refactor this and remove the comment. I'll do that now